src/js/components/molecules/main-navigation-item/main-navigation-item.ts

```diff
@@ -0,0 +1,121 @@
+import { Vue, Component, Prop, Watch } from 'vue-property-decorator';
+import { MainNavigationItemProps } from './main-navigation-item.models';
+import WithRender from './main-navigation-item.vue.html';
+
+import BrowserCapabilities from 'utils/browser-capabilities';
+
+@WithRender
+@Component({
+    name: 'MainNavigationItem',
+})
+
+export class MainNavigationItem extends Vue implements MainNavigationItemProps {
+    @Prop({ type: Boolean, required: false }) mainNavOpen?: boolean;
+
+    @Prop({ type: String, required: true }) label!: MainNavigationItemProps['label'];
+    @Prop({ type: String, required: true }) url!: MainNavigationItemProps['url'];
+    @Prop({ type: Array, required: false }) categories?: MainNavigationItemProps['categories'];
+
+    @Prop({ type: String, required: true }) itemId!: string;
+    @Prop({ type: Boolean, required: false }) isCompact?: boolean;
+    @Prop({ type: Boolean, required: true }) active!: boolean;
```

Comment:
i don’t see this property in the storybook data file, so storybook shows a console error. maybe add it to the data file?